py/dml/ctree.py

```diff
@@ -4707,6 +4712,10 @@ def ctype(self):
     def read(self):
         return self.expr.read()
     def discard(self): pass
+    def incref(self):
+        self.expr.incref()
+    def decref(self):
+        self.expr.decref()
```

Comment:
This kinda stayed. It lack of these implementations _were_ a bug, but it doesn't really matter now.